src/org/ods/orchestration/usecase/JiraUseCase.groovy

```diff
@@ -464,4 +474,131 @@ class JiraUseCase {
         return jira.getComponents(projectKey, version)
     }
 
+    protected List loadJiraSecurityVulnerabilityIssues(String issueSummary, String fixVersion,
+                                                       String jiraComponent, projectKey) {
+        if (!this.jira) {
+            logger.warn("loadJiraSecurityVulnerabilityIssues: Could *NOT* retrieve security vulnerability type issues " +
+                "because jira has invalid value.")
+            return [:]
+        }
+
+        def fields = ['assignee', 'duedate', 'issuelinks', 'status', 'summary']
+        def jql = "project = \"${projectKey}\" AND issuetype = \"Security Vulnerability\" " +
+            "AND fixVersion = \"${fixVersion}\" " +
+            "AND component = \"${jiraComponent}\" " +
+            "AND summary ~ \"${issueSummary}\" "
+
+        def jqlQuery = [
+            fields: fields,
+            jql: jql,
+            expand: []
+        ]
+
+        return jira.getIssuesForJQLQuery(jqlQuery) ?: []
+    }
+
+    List createSecurityVulnerabilityIssues(List aquaCriticalVulnerabilityRepos) {
+        def securityVulnerabilityIssueKeys = [];
+        try {
+            for (def repo : aquaCriticalVulnerabilityRepos) {
+                def jiraComponentId = getJiraComponentId(repo)
+                for (def vulnerability : repo.data.openshift.aquaCriticalVulnerability) {
+                    def vulerabilityMap = vulnerability as Map
+                    def issueKey = createOrUpdateSecurityVulnerabilityIssue(
+                        vulerabilityMap.name,
+                        jiraComponentId,
+                        buildSecurityVulnerabilityIssueDescription(
+                            vulerabilityMap,
+                            repo.data.openshift.gitUrl,
+                            repo.data.openshift.gitBranch,
+                            repo.data.openshift.repoName,
+                            repo.data.openshift.nexusReportLink))
+                    securityVulnerabilityIssueKeys.add(issueKey)
+                }
+            }
+        } catch (JiraNotPresentException e) {
```

Comment:
Yo don't need to catch thge exception, you have to check the Jira before to create the issue like in loadJiraSecurityVulnerabilityIssues